src/app/containers/README.md

```diff
@@ -0,0 +1,7 @@
+This repo follows the container component pattern.
```

Comment:
I don't think we should separate out containers - it introduces ambiguity and reduces maintainability. It would be clearer if they are next to the component, in its specific directory.